src/main/java/core/basesyntax/dao/FruitDaoCsvImpl.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.FruitTransaction;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitDaoCsvImpl implements FruitDao {
+    public static final String FILE_NAME = "src/main/resources/fruits.csv";
+    private static final String FILE_REPORT = "src/main/resources/report.csv";
```

Comment:
```suggestion
    private static final String INPUT_FILE_NAME = "src/main/resources/fruits.csv";
    private static final String OUTPUT_FILE_NAME = "src/main/resources/report.csv";
```
better move these constants to Main class, and pass them as arguments where needed